packages/editor/src/store/effects/reusable-blocks.js

```diff
@@ -61,30 +61,35 @@ export const fetchReusableBlocks = async ( action, store ) => {
 		return;
 	}
 
-	let result;
-	if ( id ) {
-		result = apiFetch( { path: `/wp/v2/${ postType.rest_base }/${ id }` } );
-	} else {
-		result = apiFetch( { path: `/wp/v2/${ postType.rest_base }?per_page=-1` } );
-	}
-
 	try {
-		const reusableBlockOrBlocks = await result;
-		dispatch( receiveReusableBlocksAction( map(
-			castArray( reusableBlockOrBlocks ),
-			( post ) => {
-				const parsedBlocks = parse( post.content.raw );
-				return {
-					reusableBlock: {
-						id: post.id,
-						title: getPostRawValue( post.title ),
-					},
-					parsedBlock: parsedBlocks.length === 1 ?
-						parsedBlocks[ 0 ] :
-						createBlock( 'core/template', {}, parsedBlocks ),
-				};
+		let posts;
+
+		if ( id ) {
+			posts = [ await apiFetch( { path: `/wp/v2/${ postType.rest_base }/${ id }` } ) ];
+		} else {
+			posts = await apiFetch( { path: `/wp/v2/${ postType.rest_base }?per_page=-1` } );
+		}
+
+		const results = compact( map( posts, ( post ) => {
+			if ( post.status === 'trash' ) {
```

Comment:
My thought was that having it this way would allow for some advanced functionality, e.g. marking a reusable block as `private` means that it becomes an _editor only_ block.

After considering it some more, I think you're right. Let's keep it simple for now and we can enable more advanced functionality in future releases if needed.

I've updated the check.